README.md

```diff
@@ -40,23 +40,25 @@ What is included in the software unit-test? What is not? Fill this table.
 |---------------------------|---------------|---
 Battery Data-accuracy       | No            | We do not test the accuracy of data
 Computation of maximum      | Yes           | This is part of the software being developed
-Off-the-shelf PDF converter | _enter Yes/No | _enter reasoning
-Counting the breaches       | _enter Yes/No | _enter reasoning
-Detecting trends            | _enter Yes/No | _enter reasoning
-Notification utility        | _enter Yes/No | _enter reasoning
+Off-the-shelf PDF converter | No            | This is a third party software, we can only ensure data we are sending is correct
+Counting the breaches       | Yes           | This is part of the software being developed
+Detecting trends            | Yes           | This is part of the software being developed
+Notification utility        | No            | Only Notification generated for new report available is tested, utility is not tested
 
-### List the Test Cases
+### List the Test Cases 
 
 Write tests in the form of `<expected output or action>` from `<input>` / when `<event>`
 
 Add to these tests:
 
 1. Write minimum and maximum to the PDF from a csv containing positive and negative readings
-1. Write "Invalid input" to the PDF when the csv doesn't contain expected data
-1. _enter a test
-1. _enter a test
-
-(add more)
+2. Write "Invalid input" to the PDF when the csv doesn't contain expected data
+3. Write number of breaches to the PDF from a csv conatining readings when threshold is crossed
+4. Write date&time to the PDF from a csv conatining readings when continuous upward trend is seen
+5. Write "Notification sent to [recipient name]" to the console when new PDF report available
+6. Write "Error Opening File" to the PDF when CSV file not accessible in server 
```

Comment:
"Console"  is it a dependency or your assumption

Reply:
it is an assumption to test if the notification send function is being called or not. mostly for testing purposes